Dropdown/Dropdown.js

```diff
@@ -0,0 +1,200 @@
+/**
+ * Agate styled Header.
+ *
+ * @example
+ * <Dropdown
+ * 		defaultSelected={2}
+ *		inline
+ *		title="Dropdown"
+ * >
+ *   {['Option 1', 'Option 2', 'Option 3', 'Option 4']}
+ * </Dropdown>
+ *
+ * @module agate/Dropdown
+ * @exports Dropdown
+ */
+
+import React from 'react';
+import kind from '@enact/core/kind';
+import Changeable from '@enact/ui/Changeable';
+import Group from '@enact/ui/Group';
+import Icon from '@enact/agate/Icon';
+import Item from '@enact/agate/Item';
+import Toggleable from '@enact/ui/Toggleable';
+import Transition from '@enact/ui/Transition';
+import PropTypes from 'prop-types';
+import compose from 'ramda/src/compose';
+import Skinnable from '../Skinnable';
+import SpotlightContainerDecorator from '@enact/spotlight/SpotlightContainerDecorator';
+import Scroller from '@enact/ui/Scroller';
+import componentCss from './Dropdown.module.less';
+
+const ContainerDiv = SpotlightContainerDecorator({enterTo: 'last-focused'}, 'div');
+const isSelectedValid = ({children, selected}) => Array.isArray(children) && children[selected] != null;
+
+/**
+ * A stateless Dropdown component.
+ *
+ * @class DropdownBase
+ * @memberof agate/Dropdown
+ * @ui
+ * @public
+ */
+const DropdownBase = kind({
+	name: 'Dropdown',
+
+	propTypes: /** @lends agate/Dropdown.Dropdown.prototype */ {
+		/*
+		 * The selections for Dropdown
```

Comment:
Should we mention that the title will be the first item if no items are selected (which seems odd to me, is that per the design)?